### PR TITLE
[TF:TRT] Update test helpers to handle bool tensors and weights

### PR DESCRIPTION
**Pull request by drivanov**
Enable the handling of bool tensors for TF-TRT test helper subroutines.

### CODE REVIEW

**Comment by drivanov** [comment]
@bixia1 : This is a substitute for [PR#54452](https://github.com/tensorflow/tensorflow/pull/54452) and I need it for converters for LogicalNot and other logical operators, which are almost ready.

File: tensorflow/compiler/tf2tensorrt/convert/utils.cc [hunk 1]
```diff
@@ -157,6 +161,10 @@ Status TfTypeToTrtType(DataType tf_type, nvinfer1::DataType* trt_type) {
     case DT_INT32:
       *trt_type = nvinfer1::DataType::kINT32;
       break;
+    // TODO add #ifdef bool is available
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/utils.cc:164]: Do you need to handle this in this PR?
If you will handle this is a follow up PR, please fix the format of the TODO description:

// TODO(your username):  end with a period.

**Review by bixia1 — APPROVED** [review]

**Comment by drivanov** [comment]
@bixia1 : All checks have passed. Could you, please, merge this one.

**Comment by drivanov** [comment]
@bixia1 : I'm fully prepared to submit a PR for `LogicalNot`, but it would be better if this one was merged.

**Comment by drivanov** [comment]
@gbaned: @bixia1 already approved these changes, but she is still mentioned as 1 pending reviewer.
Could you, please, fix that?

**Review by bixia1 — APPROVED** [review]

**Comment by drivanov** [comment]
@gbaned : I am sorry, but @bixia1 already approved these changes twice.

**Review by bixia1 — APPROVED** [review]

**Comment by bixia1** [comment]
@drivanov this test fail, see
[log](https://github.com/tensorflow/tensorflow/files/8265934/boolpr.log)

**Comment by drivanov** [comment]
> @drivanov this test fail, see [log](https://github.com/tensorflow/tensorflow/files/8265934/boolpr.log)

Fixed.

**Review by bixia1 — APPROVED** [review]

**Comment by bixia1** [comment]
This PR is already merged, see [here](https://github.com/tensorflow/tensorflow/commit/06833608a4eabdcd42133efedce009235df2f792) don't understand why the status shown here is not correct.

**Comment by gbaned** [comment]
Seems auto-merge is not happening but the changes are merged into master now, so we can close this. Thank you for the PR.

**Pull request closed without merging**